src/css/base.scss

```diff
@@ -2517,3 +2517,11 @@ textarea::placeholder {
         }
     }
 }
+
+.dc__high-light-text {
```

Comment:
Any reason for  high-light and not highlight

Reply:
done